source/test/TestApp/AutoGenTests/AutoGenProfilesTestData.cpp

```diff
@@ -3,126 +3,156 @@
 #include "TestApp.h"
 #include "AutoGenProfilesTests.h"
 #include "XAsyncHelper.h"
-#include <playfab/PlayFabClientAuthApi.h>
-#include <playfab/PlayFabClientApi.h>
-#include <playfab/PlayFabProfilesApi.h>
-#include <playfab/PlayFabAdminApi.h>
-#include <playfab/PlayFabAuthenticationAuthApi.h>
-#include <playfab/PlayFabClientDataModels.h>
 
 namespace PlayFabUnit
 {
 
-void AutoGenProfilesTests::FillPlayFabProfilesGetGlobalPolicyRequest( PlayFab::ProfilesModels::GetGlobalPolicyRequest* request )
+using namespace PlayFab::Wrappers;
+
+#pragma region GetGlobalPolicy
+
+void AutoGenProfilesTests::FillGetGlobalPolicyRequest(PFProfilesGetGlobalPolicyRequestWrapper<>& request)
 {
-    // TODO: debug  test
-    PlayFab::JsonDocument inputJson;
-    inputJson.Parse( "{}" );
-    request->FromJson(inputJson);
+    // TODO: debug Failing test
+    // Example Request: "{}"
+    UNREFERENCED_PARAMETER(request); // TODO
 }
 
-HRESULT AutoGenProfilesTests::ValidatePlayFabProfilesGetGlobalPolicyResponse( PlayFabProfilesGetGlobalPolicyResponse* result )
+HRESULT AutoGenProfilesTests::ValidatePFProfilesGetGlobalPolicyResponse(PFProfilesGetGlobalPolicyResponse* result)
 {
-    // result.permissions = PlayFabProfilesEntityPermissionStatement const* const*;
+    // result.permissions = PFProfilesEntityPermissionStatement const* const*;
     // result.permissionsCount = uint32_t;
 
     UNREFERENCED_PARAMETER(result);
     return S_OK;
 }
 
-void AutoGenProfilesTests::FillPlayFabProfilesGetEntityProfileRequest( PlayFab::ProfilesModels::GetEntityProfileRequest* request )
+#pragma endregion
+
+#pragma region GetProfile
+
+void AutoGenProfilesTests::FillGetEntityProfileRequest(PFProfilesGetEntityProfileRequestWrapper<>& request)
 {
-    // TODO: debug  test
-    PlayFab::JsonDocument inputJson;
-    inputJson.Parse( "{ \"Entity\": { \"Id\": \"1234567787392\", \"Type\": \"title_player_account\", \"TypeString\": \"title_player_account\" }}" );
-    request->FromJson(inputJson);
+    // TODO: debug Failing test
+    // Example Request: "{ \"Entity\": { \"Id\": \"1234567787392\", \"Type\": \"title_player_account\", \"TypeString\": \"title_player_account\" }}"
+    PFEntityKeyWrapper<> entity{};
+    entity.SetId("B64BE91E5DBD5597");
+    entity.SetType("title_player_account");
```

Comment:
>title_player_account [](http://example.com/codeflow?start=20&length=20)

prefer the const "PFTitlePlayerEntityType" defined in PFTitlePlayer.h

Reply:
looks like there are several spots in this file to update

Reply:
Oh good. I'll include that in the next PR